manim/utils/space_ops.py

```diff
@@ -412,24 +636,30 @@ def shoelace_direction(x_y):
     return "CW" if area > 0 else "CCW"
 
 
-def cross2d(a, b):
+def cross2d(a: int, b: int) -> float:
     if len(a.shape) == 2:
         return a[:, 0] * b[:, 1] - a[:, 1] * b[:, 0]
     else:
         return a[0] * b[1] - b[0] * a[1]
 
 
-def earclip_triangulation(verts, ring_ends):
-    """
-    Returns a list of indices giving a triangulation
-    of a polygon, potentially with holes
+def earclip_triangulation(verts: tuple, ring_ends: list) -> list:
```

Comment:
You said in the docs it's a tuple and a numpy array, but here that it is a list, which is it?

Reply:
vetrs is a numpy array, rings_ends is a list and the output was a list.
This documentation was already given so I just formatted it. I don't know what the correct typings are for this, so I will need some help for this function.
I changed the verts to `np.ndarray`.